azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/_status_logger.py

```diff
@@ -0,0 +1,52 @@
+# -------------------------------------------------------------------------
```

Comment:
Nit: I would move both status and diagnostic loggers into a `diagnostics` folder. Same with the tests.

Reply:
Can do.